src/components/icons/StartButtonIcon.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+/**
+ * The proptypes for {@link StartButtonIcon}.
+ */
+export interface StartButtonIconProps {
+  className?: string;
+}
+
+/**
+ * An icon to display in the start button on the start page of a service.
+ */
+export const StartButtonIcon: React.FunctionComponent<StartButtonIconProps> = (
+  props: StartButtonIconProps
+): JSX.Element => {
+  const { className } = props;
+
+  // This SVG is sourced from `govuk-frontend@3.3.0`. See
+  // https://github.com/alphagov/govuk-frontend/blob/c473c7ec3123e220d83221b9fd12e216f71bda84/package/govuk/components/button/template.njk#L27
+  // for the source and
+  // https://github.com/alphagov/govuk-frontend/blob/c473c7ec3123e220d83221b9fd12e216f71bda84/LICENSE.txt
+  // for the licence.
+  return (
+    <svg
+      className={classNames("govuk-button__start-icon", className)}
```

Comment:
```suggestion
      className={className}
```

We should pass the class in from `Button`, or import the SCSS for styling it in here. I prefer the former.

Reply:
This isn't resolved.